docs/overview/index.md

```diff
@@ -5,18 +5,40 @@ slug: /
 
 # What is Cradl?
 
-Cradl is the platform where developers can **create custom machine learning models for reading document data**. Use your data to build your own tailor-made models for extracting structured data from semi-structured documents. Define, train and deploy your model on Cradl - **no machine learning expertise required.**
+Cradl is a platform that enables developers to build their own document data capture APIs with deep learning.
 
-![How Cradl works illustration](/img/how-cradl-works.png)
+Our approach to data capture is quite different than most, and is centered around a single design goal:
 
-## Structured data extraction
+:::info Design Philosophy
+Be the easiest way for developers to build _customized data capture APIs_ powered by _deep learning models_.
+:::
```

Comment:
I think it's weird that the continuation sentence is placed in an info card here. Imo, Info cards are additional / supplementary information and not meant to be a continuation of previous paragraph, ie. it should be possible to have a coherent reading experience without needing to read all the info cards. I suggest completing the sentence outside the info card, but keep a condensed version as the info card.

Reply:
I dont' have a strong opinion on this, so I'm perfectly fine with something like what you describe here.